web_fragments/test_utils/views.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Simple fragment view for testing.
+"""
+
+from __future__ import absolute_import, unicode_literals
+
+from web_fragments.fragment import Fragment
+from web_fragments.test_utils import TEST_CSS, TEST_HTML, TEST_JS
+from web_fragments.views import FragmentView
+
+TEST_FRAGMENT_VIEW_NAME = 'test_fragment_view'
+
+
+class TestFragmentView(FragmentView):
+    """
+    Simple fragment view for testing.
+    """
+
+    def render_fragment(self, request, **kwargs):
+        """
+        Returns a simple fragment
+        """
+        fragment = Fragment(TEST_HTML)
+        fragment.add_javascript(TEST_JS)
+        fragment.add_css(TEST_CSS)
+        return fragment
```

Comment:
@andy-armstrong Agree with @pmitros here - this would be very nice to have closer to actual users. Maybe an "examples" folder?